src/main/java/stringCalculator/domain/Number.java

```diff
@@ -0,0 +1,39 @@
+package stringCalculator.domain;
+
+import java.util.Objects;
+
+public class Number {
+  private int number;
+
+  public Number(String numberString) {
+    if ( numberString == null || "".equals(numberString))
+      numberString = "0";
```

Comment:
if 문의 내부가 한 문장일 경우에도 중괄호로 둘러싸는게 컨벤션이오니 구글 자바 가이드 문서를 참고하시면 좋을 것 같아요!
```
4.1.1 Braces are used where optional
Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement
```
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

 ```suggestion
    if ( numberString == null || "".equals(numberString)) {
        numberString = "0";
    }
```

Reply:
넵! 감사합니다! 다음 미션때부터 적용해볼게요!